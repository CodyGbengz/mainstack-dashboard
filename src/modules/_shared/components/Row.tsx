import styled from 'styled-components';

export  const Row = styled.div`
  justify-content: space-between;
  flex-flow: row nowrap;
  display: flex;
`;