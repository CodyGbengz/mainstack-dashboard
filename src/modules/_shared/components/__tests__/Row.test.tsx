import { render } from '@testing-library/react';
import { Row } from '../Row';

describe('COMPONENT TEST: Row', () => {
  const setup = () => render(<Row />);

  test('Should match snapshot', () => {
    const { container } = setup();
    expect(container.parentElement).toMatchSnapshot();
  });
});
