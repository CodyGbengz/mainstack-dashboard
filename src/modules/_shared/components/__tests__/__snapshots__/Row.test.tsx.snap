// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`COMPONENT TEST: Row > Should match snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-bdfCkl cBIeWz"
    />
  </div>
</body>
`;
